doc/releases/changelog-dev.md

```diff
@@ -148,13 +148,73 @@
   operation on our quantum circuits for both qubit and CV devices.
   [(#1829)](https://github.com/PennyLaneAI/pennylane/pull/1829)
 
+* A function for computing generalized parameter shift rules for generators'
+  whose eigenvalue frequency spectrum is known is available as `qml.gradients.get_shift_rule`.
+  Given a generator's frequency spectrum of `R` unique frequencies, `qml.gradients.get_shift_rule`
+  returns the parameter shift rules to compute expectation value gradients of the generator's
+  variational parameter using `2R` shifted cost function evaluations. This becomes cheaper than
+  the standard application of the chain rule and two-term shift rule when `R` is less than the
+  number of Pauli words in the Hamiltonian generator.
+
+  For example, a four-term shift rule is generated for the frequency spectrum `[1, 2]`, which
+  corresponds to a generator eigenspectrum of e.g., `[-1, 0, 1]`:
+
+  ```pycon
+  >>> frequencies = (1,2)
+  >>> grad_recipe = qml.gradients.get_shift_rule(frequencies)
+  >>> grad_recipe
+  ([[0.8535533905932737, 1, 0.7853981633974483], [-0.14644660940672624, 1, 2.356194490192345],
+    [-0.8535533905932737, 1, -0.7853981633974483], [0.14644660940672624, 1, -2.356194490192345]],)
+  ```
+
+  As we can see, `get_shift_rule` returns a tuple containing a list of four nested lists for the
+  four term parameter shift rule. Each term :math:`[c_i, a_i, s_i]` specifies a term in the
+  gradient reconstructed via parameter shifts as
+
+  .. math:: \frac{\partial}{\partial\phi_k}f = \sum_{i} c_i f(a_i \phi_k + s_i).
+
+* A circuit template for time evolution under a commuting Hamiltonian utilizing generalized
+  parameter shift rules for cost function gradients is available as `qml.CommutingEvolution`.
+  [(#1788)](https://github.com/PennyLaneAI/pennylane/pull/1788)
+
+  If the template is handed a frequency spectrum during its instantiation, then `get_shift_rule`
+  is internally called to obtain the general parameter shift rules which will be used when computing
+  the gradient of the cost function with respect to `CommutingEvolution`'s :math:`t` parameter.
+  If a frequency spectrum is not handed to `CommutingEvolution`, then cost function gradients will
+  be computed in the standard manner.
```

Comment:
Does this say "if parameter-shift differentiation is used, handing frequencies to `CommutingEvolution` will create a custom shift rule, otherwise the shift rule for a decomposition of `CommutingEvolution` will be used." or does it say
"Even if differentiation is set to something different than parameter shifts, providing frequencies to `CommutingEvolution`  will change this setting for this operation, otherwise the original differentiation strategy will be kept." ?

Reply:
Great point @dwierichs, my intention was to state the former. I've updated in the latest commit.